### ISSUE TITLE
Nested Expressions/Properties

### ISSUE THREAD

**Issue by MKlauck** [convince-project/smc_storm#18]
G(p -> Finally q)

**Comment by MarcoLm993** [comment]
Hmmm, this is a non-atomic property, and also, would need to expand the model until a final state (or, alternatively, a straightforward loop in the same state)

**Comment by MKlauck** [comment]
You are right. Then let's postpone all of that further. For iFM we are anyways too late ;-)
